src/ucp/rma/flush.c

```diff
@@ -201,14 +259,24 @@ ucs_status_t ucp_ep_flush_progress_pending(uct_pending_req_t *self)
 
     ucs_assert(!(req->flags & UCP_REQUEST_FLAG_COMPLETED));
 
+    if (ep->flags & UCP_EP_FLAG_BLOCK_FLUSH) {
+        ucp_ep_flush_slow_path_resched(ep, req);
+        return UCS_OK;
+    }
+
+    ucs_assert(lane != UCP_NULL_LANE);
+    ucs_assert(!(req->send.flush.started_lanes & UCS_BIT(lane)));
```

Comment:
maybe this assert not needed - already done in ucp_ep_flush_request_change_uct_cmp
or convert to assertv